charts/ia-case-api/values.preview.template.yaml

```diff
@@ -8,8 +8,8 @@ java:
     POSTGRES_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     POSTGRES_CONNECTION_OPTIONS: ""
-    IA_CASE_DOCUMENTS_API_URL: "http://ia-case-documents-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
-    IA_CASE_NOTIFICATIONS_API_URL: "http://ia-case-notifications-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/"
+    IA_CASE_NOTIFICATIONS_API_URL: "http://ia-case-notifications-api-pr-292.service.core-compute-preview.internal/"
+    IA_CASE_DOCUMENTS_API_URL: "http://ia-case-documents-api-pr-238.service.core-compute-preview.internal/"
```

Comment:
something to revert when `ia-case-documents-api` and `ia-case-notifications-api` are released